crates/ignore/src/default_types.rs

```diff
@@ -23,7 +23,8 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("avro", &["*.avdl", "*.avpr", "*.avsc"]),
     ("awk", &["*.awk"]),
     ("bazel", &[
-        "*.bazel", "*.bzl", "*.BUILD", "*.bazelrc", "BUILD", "WORKSPACE",
+        "*.bazel", "*.bzl", "*.BUILD", "*.bazelrc", "BUILD", "MODULE.bazel", "WORKSPACE",
+        "WORKSPACE.bazel",
```

Comment:
It's hard to tell in GitHub's UI, but could you please make sure lines are wrapped to 79 columns inclusive? Thanks.

Reply:
Done!